src/genie/libs/parser/iosxr/tests/show_interface_summary.py

```diff
@@ -0,0 +1,100 @@
+import unittest
```

Comment:
We've actually moved over to the new folder based unit tests that you've already made. This file based script one is actually deprecated. Can you please remove it from the pr?

Reply:
I noticed that and will remove the file with the next commit.